lib/semantics/mod-file.cc

```diff
@@ -85,21 +88,10 @@ void ModFileWriter::Write(const Symbol &symbol) {
   auto *ancestor{symbol.get<ModuleDetails>().ancestor()};
   auto ancestorName{ancestor ? ancestor->name().ToString() : ""s};
   auto path{ModFilePath(dir_, symbol.name(), ancestorName)};
-  unlink(path.data());
```

Comment:
Do you still need the system include files?

Reply:
I've removed unistd.h and set.